apps/synapse-portal-framework/src/components/DetailsPage/DetailsPage.tsx

```diff
@@ -83,40 +71,34 @@ export default function DetailsPage(props: DetailsPageProps) {
     data: asyncJobStatus,
     isLoading,
     isSuccess,
-    error: hasError,
+    error: queryError,
   } = useGetQueryResultBundleWithAsyncStatus(queryBundleRequest)
 
   const queryResultBundle = asyncJobStatus?.responseBody
+  let row: Row | undefined
+  if (queryResultBundle?.queryResult?.queryResults?.rows) {
+    row = queryResultBundle?.queryResult.queryResults.rows[0]
+  }
+  const rowNotFound = !isLoading && !row
 
-  if (hasError) {
-    const currentLocation = location.pathname.split('/')
-    const name = decodeURI(currentLocation[currentLocation.length - 2])
+  const candidateDoiId = getCandidateDoiId({
+    portalDoiConfiguration: portalDOIConfiguration,
+    data: searchParams,
+  })
+
+  if (rowNotFound || queryError) {
+    // If the row does not exist, or the link is broken, show the generic error page.
     return (
-      <Stack
-        sx={{
-          alignItems: 'center',
-          gap: 2,
-          mt: 5,
-        }}
-      >
-        <Typography variant="headline1" gutterBottom>
-          Coming Soon!
-        </Typography>
-        <Typography variant={'smallText1'} gutterBottom>
-          This {pluralize.singular(name).toLowerCase()} is not yet available,
-          please check back soon.
-        </Typography>
```

Comment:
Removed this "Coming Soon" view that was shown when an error was encountered. It could be misleading if you followed a link to a resource that was moved or removed.